core/server/data/xml/sitemap/base-generator.js

```diff
@@ -65,7 +65,13 @@ class BaseSiteMapGenerator {
     }
 
     getLastModifiedForDatum(datum) {
-        return datum.updated_at || datum.published_at || datum.created_at;
+        if (datum.updated_at || datum.published_at || datum.created_at) {
+            const modifiedDate = datum.updated_at || datum.published_at || datum.created_at;
+
+            return moment(new Date(modifiedDate));
```

Comment:
have checked the usages in the codebase, think it's unnecessary here and below to wrap the date in additional `new Date()` constructor. [See](https://momentjs.com/docs/#/parsing/now/) explanation :wink: